sca/windows/cis_win2012r2_domainL1_rcl.yml

```diff
@@ -20,6 +20,13 @@ policy:
   references:
     - https://www.cisecurity.org/cis-benchmarks/
 
+requirements:
+  title: "Check for Windows platform"
```

Comment:
This is the title what is shown in the log message. It could be more descriptive: `Check that the Windows platform is Windows Server 2012 R2`. The same for the rest of policies for Server 2012 R2.